corpus/transform.py

```diff
@@ -491,3 +491,73 @@ def run(self):
 
         with open(str(self.out_segments), "w") as segments_outfile:
             segments_outfile.writelines(segment_file_names)
+
+
+class LexiconStrategy(enum.Enum):
+    PICK_FIRST = 0
+
+
+class ApplyLexiconToTranscriptionsJob(Job):
```

Comment:
```suggestion
class ApplyLexiconToCorpusJob(Job):
```
I suggest a renaming to make more clear that this is applied to a bliss corpus (as opposed to plain text files)